reference-tests/src/test/java/net/consensys/linea/GeneralStateReferenceTestTools.java

```diff
@@ -81,11 +81,10 @@ private static ProtocolSpec protocolSpec(final String name) {
               (testName, fullPath, stateSpec, collector) -> {
                 final String prefix = testName + "-";
                 for (final Map.Entry<String, List<GeneralStateTestCaseEipSpec>> entry :
-                    stateSpec.finalStateSpecs().entrySet().stream()
-                        .filter(e -> e.getKey().equalsIgnoreCase("London"))
```

Comment:
- This filter results in 0 test configs for some GeneralStateReferenceTests. 
- Since those are parameterized tests and if there are no tests  parameters generated, junit throws and excpection `Configuration error: You must configure at least one set of arguments for this @ParameterizedTest`. 
- This change will mitigate this exception and simply skip the non london tests.